### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by florian-vuillemot**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,12 +4,11 @@ verify_ssl = true
...
 
 [packages]
-networkx = ">=2.6"
-numpy = ">=1.21"
-openpyxl = ">=3.0"
-pandas = ">=1.3"
-sqlalchemy = ">=1.3"
-toml = ">=0.10"
+networkx = "==2.6"
+openpyxl = "==3.0.3"
```
Comment (FabienLelaquais) [inline Pipfile:8]: Is == required here??

&gt;=3.0 wasn't enough? Or maybe we're lacking a &lt;[something]

File: setup.py [hunk 2]
```diff
@@ -19,13 +19,11 @@
...
 requirements = [
-    "networkx",
-    "numpy",
-    "openpyxl",
-    "pandas",
-    "simple-websocket",
-    "sqlalchemy",
-    "toml",
+    "networkx>=2.6",
+    "openpyxl>=3.0.3",
```
Comment (FabienLelaquais) [inline setup.py:23]: Something like that, yes.

**Review by FabienLelaquais — DISMISSED** [review]
With my little comments... Thx.

File: Pipfile (hunk 1 above)
Comment (jrobinAV) [reply Pipfile:8]: I am fine with setting an exact version if it is too complicated to follow the compatibility constraints.

**Review by jrobinAV — DISMISSED** [review]

File: Pipfile (hunk 1 above)
Comment (florian-vuillemot) [reply Pipfile:8]: _Pipfile_ is for development only, for installation _setup.py_ is used.
I think, setting minimal versions as constraints in the _Pipfile_ will ensure developers to work with the minimal version that a user can have on its machine.
If we need new functionality of a package, we should update the version in the _Pipfile_ and thus the _setup.py_ and we are sure to not break the user as we did.

Comment (florian-vuillemot) [reply Pipfile:8]: For _openpyxl_ only. Check https://pypi.org/project/openpyxl/#history
We should deal with the "patch/technical/.." version. There are using it as we used the "minor".

Comment (FabienLelaquais) [reply Pipfile:8]: I guess my point was: do we know of any issues with openpyxl&gt;=3.0?

**Review by jrobinAV — APPROVED** [review]

File: Pipfile (hunk 1 above)
Comment (florian-vuillemot) [reply Pipfile:8]: Oh ^^' 
Yes, failed until the 3.0.3 with the following error:
image

**Pull request merged**